src/equinoxe/equinoxe.ml

```diff
@@ -717,6 +724,25 @@ module Make (B : Backend) : API with type 'a io = 'a B.io = struct
       with Ezjsonm.Parse_error (v, err) ->
         fail_with_parsing ~name:"Device.create" ~err ~json v
 
+    let update t ~id ?hostname ?tags () =
+      let path = Format.sprintf "devices/%s" id in
+      let fields =
+        (match hostname with
+        | Some hostname -> [ ("hostname", `String hostname) ]
+        | None -> [])
+        @
+        match tags with
+        | Some tags ->
+            let tags = List.map (fun tag -> `String tag) tags in
+            [ ("tags", `A tags) ]
+        | None -> []
```

Comment:
A small redundancy between this and `builder_to_json` :)

Reply:
Done :+1: